### PR TITLE
15768: Extend Glue Data Model to handle carrier specific plan ID

### PR DESCRIPTION
**Pull request by prem548**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [15768](https://devops.dchbx.org/redmine/issues/15768) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | n/a |
| Development Story Points | 1 |
| Peer Review Story Points | 1 |
| Ticket Author | @TreyE |

### CODE REVIEW

File: app/models/policy.rb [hunk 1]
```diff
@@ -31,6 +31,10 @@ class Policy
   field :is_active, type: Boolean, default: true
   field :hbx_enrollment_ids, type: Array
 
+# Adding field values Carrier specific
+  field :carrier_specific_plan_id, type: String, default: ""
+  field :rating_area, type: String
+  field :rating_tier, type: String
```
Comment (TreyE) [inline app/models/policy.rb]: Minor nitpick, but this should be called 'composite_rating_tier'.

**Comment by prem548** [comment]
@TreyE I updated the PR as per comments. Please let me know, if any changes necessary.

File: app/views/people/_policy_detail.html.erb [hunk 2]
```diff
@@ -51,6 +51,15 @@
...
 	  	<dd><%= p.updated_at.strftime("%m-%d-%Y %I:%M:%S %p %Z") %></dd>
+
+      <% if p.rating_area || p.carrier_specific_plan_id || p.composite_rating_tier %>
+        <dt>CarrierPlanID</dt>
+         <dd>p.carrier_specific_plan_id</dd>
+        <dt>Rating Area</dt>
+         <dd>p.rating_area </dd>
+        <dt>Composite Rating Tier </dt>
+          <dd>p.composite_rating_tier </dd>
+      <% end %>
```
Comment (TreyE) [inline app/views/people/_policy_detail.html.erb]: Almost there - need to check for each of these separately, as they don't depend on each other to be present.

**Comment by prem548** [comment]
I updated the changes suggested. @TreyE

**Pull request merged**